endpoints/calendar/tests/integration_test.js

```diff
@@ -27,3 +29,11 @@ describe('calendar/:year/:month/:day', () => {
     request.get(params, resultHandler)
   })
 })
+
+
+describe('validateParams', () => {
+  it('should return an error if a parameter could not be parsed as an Int', () => {
+    expect(normalizeParams('bogusYear', 1, 1))
+      .toEqual({ error: 'Year must be a number' })
+  })
+})
```

Comment:
This isn't an integration test. We should probably move this to it's own file, but that would require changing the test running script a bit.